src/main/resources/hudson/plugins/promoted_builds/PromotionStatusColumn/column.jelly

```diff
@@ -0,0 +1,47 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <td>
+    <j:set var="ppa" value="${it.getAction(job)}"/>
+    <j:choose>
+      <j:when test="${ppa!=null}">
+        <j:set var="pp" value="${ppa.getPromotionProcesses()}"/>
+        <j:choose>
+          <j:when test="${pp!=null}">
```

Comment:
I had forgotten that I made it return an empty list so I would need a `<j:choose>` / `<j:when>`... It is possible to manually check all other cases by configuring a view with the column and some sample jobs. I can simplify this jelly. Is that ok?